### PR TITLE
data:snehalsenapati23

### PR DESCRIPTION
**Pull request by snehalsenapati23**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3982"></a>

### CODE REVIEW

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>
Looks good to merge. Thank you for your contribution! :tada:

File: data/snehalsenapati23.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Snehal Senapati",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer | Open Source Enthusiast | HTML | CSS | ReactJS | DSA-Java | ",
+  "avatar": "https://github.com/snehalsenapati23.png"
```
Comment (Pradumnasaraf) [inline data/snehalsenapati23.json:6]:
```suggestion
  "avatar": "https://github.com/snehalsenapati23.png",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

There is a missing `,` please fix that

File: data/snehalsenapati23.json (hunk 1 above)
Comment (snehalsenapati23) [reply data/snehalsenapati23.json]: added ,

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**